packages/cli/src/services/user.service.ts

```diff
@@ -57,9 +60,9 @@ export class UserService {
 			withScopes?: boolean;
 		},
 	) {
-		const { password, updatedAt, apiKey, authIdentities, ...rest } = user;
+		const { updatedAt, authIdentities, ...rest } = user.toJSON();
```

Comment:
Should the `.toJSON` be named something like `.withoutSensitiveFields()` to make it more explicit what it does?

Reply:
I think we've been using `toJSON` to address the lack of response DTOs leading to these fields getting send to the frontend whenever these user objects are serialized.
The better solution here would to be have explicit response DTOs, and have linting rules to prevent people from returning database objects in route handlers.